command/eval_status.go

```diff
@@ -203,9 +204,21 @@ func (c *EvalStatusCommand) Run(args []string) int {
 		statusDesc = eval.Status
 	}
 
+	// Format eval timestamps
+	var formattedCreateTime, formattedModifyTime string
+	if verbose {
+		formattedCreateTime = formatUnixNanoTime(eval.CreateTime)
+		formattedModifyTime = formatUnixNanoTime(eval.ModifyTime)
+	} else {
+		formattedCreateTime = prettyTimeDiff(time.Unix(0, eval.CreateTime), time.Now())
+		formattedModifyTime = prettyTimeDiff(time.Unix(0, eval.ModifyTime), time.Now())
+	}
+
 	// Format the evaluation data
 	basic := []string{
 		fmt.Sprintf("ID|%s", limit(eval.ID, length)),
+		fmt.Sprintf("Create Time|%s", formattedCreateTime),
+		fmt.Sprintf("Modify Time|%s", formattedModifyTime),
```

Comment:
Pointing out that for old evals this will print an empty string, which may be ok but odd UX.

Reply:
Agreed it's not the greatest UX. This behavior is consistent with `nomad alloc status`, although it's unlikely at this point that there are old allocs w/o a CreateTime. Going to think about it, will most likely leave as is but add a note in the docs.

Reply:
Leaving it as is sounds good to me if there's prior art already that does this.